docs/UserGuide.adoc

```diff
@@ -746,7 +874,7 @@ The list of items ordered by dining customers at a table
 The list of items available to be ordered by dining customers at the restaurant
 
 [[bill]] Bill::
-The amount of money that dining customers have to pay based on their order
+The amount of money that dining customers have to pay based on their orders
```

Comment:
Nope, "order" is correct. Refer to the definition of order above this

Reply:
okk